packages/x-tree-view/src/internals/utils/tree.ts

```diff
@@ -22,7 +22,7 @@ const getLastNavigableItemInArray = (
 export const getPreviousNavigableItem = (
```

Comment:
Side note: Would be awesome if we could strongly battle test those methods to avoid bugs
Because we probably don't want 30 tests on `ArrowDown`, 30 tests on `ArrowUp` etc... to check edge-cases of this method.